lib/object/has-property.js

```diff
@@ -5,22 +5,39 @@
  * @author Eric Satterwhite
  **/
 
-module.exports = function hasOwnProperty(obj, prop) {
-  return Object.prototype.hasOwnProperty.call(obj, prop)
+module.exports = function hasOwnProperty(obj, string = '', sep = '.') {
```

Comment:
I don't like this named the same as the prototype name.  It should be `hasProperty` to match the filename anyway.

Reply:
Updated!